packages/v8-deopt-parser/test/parseV8Log.test.js

```diff
@@ -1,359 +1,363 @@
 import test from "tape";
 import {
-	runParser,
+	runParserNonStream,
+	runParserStream,
 	writeSnapshot,
 	validateEntry,
 	repoRoot,
 	repoFileURL,
 } from "./helpers.js";
 import { expectedICSLogs } from "./constants.js";
 
-test("runParser(adders.v8.log)", async (t) => {
-	const logFileName = "adders.v8.log";
-	const result = await runParser(t, logFileName);
-
-	t.equal(result.codes.length, 16, "Number of codes");
-	t.equal(result.deopts.length, 7, "Number of deopts");
-	t.equal(result.ics.length, 33, "Number of ics");
-
-	validateEntry(t, "Matching ICS Entry", result.ics, expectedICSLogs.adders);
-
-	validateEntry(t, "Matching Deopt entry", result.deopts, {
-		type: "deopts",
-		id: "331",
-		functionName: "addObjects",
-		file: repoRoot("examples/simple/adders.js"),
-		line: 137,
-		column: 28,
-		severity: 3,
-		updates: [
-			{
-				timestamp: 496191,
-				bailoutType: "eager",
-				deoptReason: "wrong call target",
-				optimizationState: "optimized",
-				inlined: false,
-				severity: 3,
-				inlinedAt: undefined,
-			},
-		],
+for (const runParser of [runParserNonStream, runParserStream]) {
```

Comment:
For posterity, I manually diffed this file in VSCode and it just adds the `for` loop around the existing tests. The tests are the same. GitHub diff is just showing a more complicated diff than necessary.

Great work!